docker-compose.debug.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file is for **development** setup. For production, refer to
+# docker-compose.production.yml.
+version: '3'
+services:
+  server:
+    command: debug_server
+    ports:
+      - "5678:5678"
+      - "5000:5000"
```

Comment:
How about 85f9598? This way you would switch to debugging by running:

```bash
docker-compose stop server && docker-compose run --rm --service-ports server debug && docker-compose start server
```
and attaching via VSCode. Once you're done, a Ctrl+C will get you back to your usual reloading, non-debuggable dev server.

Reply:
👍 

I think it's time to introduce `bin/dev` to the OSS repo too, and then we can do `bin/dev debug` to avoid the copy pasting.

I'll take care of this.